crates/rnote-engine/src/store/keytree.rs

```diff
@@ -69,6 +70,18 @@ impl KeyTree {
     pub(crate) fn clear(&mut self) {
         *self = Self::default()
     }
+
+    pub(crate) fn get_tree(&self) -> &rstar::RTree<KeyTreeObject, rstar::DefaultParams> {
+        &self.0
+    }
+
+    pub fn get_bounds(&self) -> AABB<[f64; 2]> {
```

Comment:
RTree's AABB type should not be exposed outside of the module and instead be converted to parry's Aaab